### PR TITLE
Remove Fax functionality

### PR DESCRIPTION
**Pull request by eskie-aloware**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "signalwire/signalwire",
+  "name": "aloware/signalwire",
```
Comment (amir-the-h) [inline composer.json:2]: Do we need to change this? Considering we might want to push back the codes into the original repo.

File: src/Rest/Client.php [hunk 2]
```diff
@@ -47,33 +40,4 @@ private function _checkEnv(String $key) {
...
-   *
-   * @param \Twilio\Domain $domain Domain that contains the version
-   * @return \Twilio\Rest\Fax\V1 V1 version of Fax
-   */
-  public function __construct(Fax $domain) {
-      parent::__construct($domain);
-      $this->version = '2010-04-01/Accounts/' . $domain->client->username;
-  }
-}
+}
```
Comment (amir-the-h) [inline src/Rest/Client.php:43]: Please add an empty new line at the end of the file.

**Review by amir-the-h — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (eskie-aloware) [reply composer.json:2]: oh i see.. okay ill revert it..

**Comment by eskie-aloware** [comment]
applied changes

**Review by amir-the-h — APPROVED** [review]

**Pull request merged**